app/models/in_person_enrollment.rb

```diff
@@ -22,6 +22,28 @@ class InPersonEnrollment < ApplicationRecord
   before_save(:on_status_updated, if: :will_save_change_to_status?)
   before_create(:set_unique_id, unless: :unique_id)
 
+  def self.pending_established(start_interval, end_interval)
```

Comment:
I looked at the calls below before seeing this method definition and it wasn't immediately clear to me what the arguments were. Maybe including "between" or "in range" in the method name could help clarify?